Formula/rure.rb

```diff
@@ -1,8 +1,8 @@
 class Rure < Formula
-  desc "C API for RUst's REgex engine"
```

Comment:
I think the capitalisation was to explain why it's called `rure`.

Reply:
haha, now i see. I will revert my change.